docs/occlum_mount.md

```diff
@@ -26,3 +26,14 @@ Step 2: Mount the secure FS at the newly created mount point
 occlum mount [--sign-key <key_path>] [--sign-tool <tool_path>] <path>
 ```
 After mounting the secure FS successfully, you can access and manipulate the FS from the mount point as easy as regular Linux FS.
+
+## Troubleshooting
+
+* FUSE lookup error:
+    - Error message:
+    ```
+    fuse: device not found, try 'modprobe fuse' first
+    Error: Os { code: 2, kind: NotFound, message: "No such file or directory" }
+    ```
+    - Root cause: The docker container is not allowed to use FUSE by default, check out the discussion [here](https://github.com/docker/for-linux/issues/321).
+    - Solution: Run the docker container with additional flags: `--cap-add SYS_ADMIN --device /dev/fuse`.
```

Comment:
I think it would be better to rewrite the troubleshooting as a prerequisites section at the very beginning of this doc. This is because >80% of users would run Occlum inside a Docker and most of them would not give `--cap-add SYS_ADMIN --device /dev/fuse` or `--privileged` flags to Docker unless they have to do so.  So by describing the requirement upfront, we prevent the users from encountering an unhappy situation that is likely to happen.

Reply:
make sense, I will update this commit